src/js/filter.ts

```diff
@@ -44,18 +44,30 @@ const shouldBeRendered = (
       )
     );
 
-  const scopeSelected = getSelected(".filter--scope :checked");
+  const scopeSelected = getSelected(".scope :checked");
   const isScope = entry["report_magnitude"]
     .split(",")
     .some((scope) => scopeSelected.has(scope));
+  const policySelected = getSelected(".policy-change :checked");
+  const isPolicy = entry["report_type"]
+    .split(",")
+    .some((policy) => policySelected.has(policy));
+  const landSelected = getSelected(".land-use :checked");
+  const isLand = entry["land_uses"]
+    .split(",")
+    .some((land) => landSelected.has(land));
+  const stageSelected = getSelected(".implementation-stage :checked");
+  const isStage = entry["report_status"]
+    .split(",")
+    .some((stage) => stageSelected.has(stage));
```

Comment:
I think I will make xSelected and isX into one line. And add a comment to describe what is going on.